app/payment-link-v2/amount/get-index.controller.test.js

```diff
@@ -52,7 +52,7 @@ describe('Amount Page - GET controller', () => {
       res = {}
       controller(req, res)
 
-      expect(responseSpy.called).to.equal(true)
+      sinon.assert.called(responseSpy)
       expect(mockResponses.response.args[0]).to.include(req)
       expect(mockResponses.response.args[0]).to.include(res)
       expect(mockResponses.response.args[0]).to.include('amount/amount')
```

Comment:
```suggestion
      sinon.assert.calledWith(responseSpy, req, res, 'amount/amount')
```

Reply:
Good idea - done in a bunch of places.
This looks a lot cleaner.